gateway/gradle.properties

```diff
@@ -0,0 +1,34 @@
+rootProject.name=gateway
+profile=dev
+
+# Build properties
+node_version=14.16.1
+yarn_version=1.22.10
+
```

Comment:
What is the reason why hibernate is not present in the gradle.properties for the gateway (but present for artemis?)

Reply:
We do not have a db connection in the gateway, neither db entities and repositories we work with, therefore it is not used and not needed.